config/locales/en.yml

```diff
@@ -225,6 +225,9 @@ en:
       transparency:
         one: Transparency data
         other: Transparency data
+      world_location_news_article:
```

Comment:
I think the history notice reads better as "This news article...". I don't think the user needs to understand the difference between news article and world location news article.